src/t_hash.c

```diff
@@ -575,6 +645,32 @@ static int _hashZiplistEntryValidation(unsigned char *p, void *userdata) {
     return 1;
 }
 
+/* callback for to check the listpack doesn't have duplicate records */
+static int _hashListpackEntryValidation(unsigned char *p, void *userdata) {
+    unsigned char buf[LP_INTBUF_SIZE];
+
+    struct {
+        long count;
+        dict *fields;
+    } *data = userdata;
+
+    /* Odd records are field names, add to dict and check that's not a dup */
```

Comment:
the code (comment) was changed (fixed) after you copied it.
see bf92000e2dc33d484c73eaaa0af5cc47acf81b82
maybe you should check for other recent changes especially in ziplist.c and t_hash.c
```suggestion
    /* Even records are field names, add to dict and check that's not a dup */
```